Form/EventListener/ResizePolyFormListener.php

```diff
@@ -84,7 +85,12 @@ public function __construct(array $prototypes, array $options = array(), $allowA
                 $defaultType = $type;
             }
 
-            $typeKey = $type instanceof FormTypeInterface ? $type->getName() : $type;
+            $typeKey = $type;
+
+            if ($type instanceof FormTypeInterface) {
```

Comment:
I've moved the deprecation I added (to the Resize listener) to the PolyCollectionType::buildPrototypes method. Seems to be the best place to put it. https://github.com/infinite-networks/InfiniteFormBundle/commit/ef793944d50fa9aef2dc7b26be815d95c27c7e39